.golangci-warnings.yml

```diff
@@ -0,0 +1,56 @@
+run:
+  timeout: 5m
+  tests: true
+
+linters:
+  disable-all: true
+  enable:
+    - staticcheck
+    - structcheck
+    - typecheck
+    - varcheck
+    - deadcode
+    - gosimple
+    - unused
+
+severity:
+  default-severity: warning
+
+issues:
+  # use these new lint checks on code since #2574
+  new-from-rev:  99ab66fdadcd77fef4048bb2aa86cae1f06c6e08
```

Comment:
You'll want to remove this if you have already fixed all the linting issues in this codebase — this references a commit on the go-algorand repo that I used to ignore all lint issues before this commit. (Because it was too much work to fix all the issues in the code before that date)

Reply:
oh I see 99ab66fdadcd77fef4048bb2aa86cae1f06c6e08 is a commit on indexer, well, never mind

Reply:
I should update the description.